### PR TITLE
Pull Request for Restaurant_page project

### PR DESCRIPTION
**Pull request by abouhid**
This project consists of developing a restaurant page using the recently acquired knowledge in webpack. 
The PR is necessary for the approval and submission of the project.
Using mostly Javascript, I was able to insert elements and features. 
The project is made of different modules using webpack and responsiveness was not considered.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "restaurant_page",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (addod19) [inline package.json]: We are missing the following scripts
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
+    "file-loader": "^6.2.0",
+    "style-loader": "^2.0.0",
+    "stylelint": "^13.3.3",
+    "stylelint-config-standard": "^20.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.17.2",
+    "webpack": "^5.4.0",
```
Comment (addod19) [inline package.json:33]: [Optional] - Let us kindly add ` webpack-dev-server`

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @abouhid, great job on this project :+1: The application looks great :slightly_smiling_face: The App works as expected :smile: However, there are a few issues we need to address before proceeding. Kindly check the comments below: :fireworks: 

- PR needs to be improved. Kindly edit the PR and add the necessary title and summary of the project
- README file is missing instructions about how to clone and run the project. Let us add it

Once these are addressed, kindly submit another code review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import './styles/style.css';
+
+import mainPage from './tabs/main-page';
+import menuPage from './tabs/menu-page';
+import infoPage from './tabs/info-page';
+
+function component() {
+  const content = document.getElementById('content');
```
Comment (trekab) [inline src/index.js:8]: __[Optional]__ It is a requirement for this project that you use ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)). Make use of the `arrow` function in this case.

**Review by trekab — APPROVED** [review]
### Approved! :tada: 
Well done implementing the suggested changes from the last review. Keep up the good work!

__N.B:__ Check out the optional change you could make before merging your PR in the code below.

_Happy coding!_ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Pull request merged**